images/linux/scripts/installers/azpowershell.sh

```diff
@@ -8,90 +8,37 @@
 source $HELPER_SCRIPTS/document.sh
 source $HELPER_SCRIPTS/os.sh
 
-# Install Azure CLI (instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+# List of version
 if isUbuntu20 ; then
     latestVersion=$(pwsh -Command '(Find-Module -Name Az).Version')
-    modulePath="/usr/share/az_$latestVersion"
-    echo "Save Az Module ($latestVersion) to $modulePath"
-    pwsh -Command "Save-Module -Name Az -LiteralPath $modulePath -RequiredVersion $latestVersion -Force"
-
-    # Run tests to determine that the software installed as expected
-    echo "Testing to make sure that script performed as expected, and basic scenarios work"
-    if ! pwsh -Command "\$env:PSModulePath = '${modulePath}:' + \$env:PSModulePath
-        if ( -not (Get-Module -ListAvailable -Name Az.Accounts)) {
-            Write-Host 'Az Module was not installed'
-            exit 1
-        }"; then
-        exit 1
-    fi
-
-    # Document what was added to the image
-    DocumentInstalledItem "Az Module ($latestVersion)"
-    exit 0
+    versions=($latestVersion)

```

Comment:
why we need two variables instead just
```
# Take the latest version only for Ubuntu20
versions=$(pwsh -Command '(Find-Module -Name Az).Version')
```

Reply:
@miketimofeev , I don't think that installing the latest version is good idea here.
For example, 3.5.0 and 3.8.0 contain breaking changes. If we install the latest only, it will break customers